core/model/src/main/java/org/eclipse/sw360/antenna/api/exceptions/ExecutionException.java

```diff
@@ -13,19 +13,19 @@
 /**
  * Used when a step of the Antenna execution workflow unexpectedly fails.
  */
-public class AntennaExecutionException extends RuntimeException {
+public class ExecutionException extends RuntimeException {
```

Comment:
Some might confuse the name `ExecutionException` with the `ExecutionException` from `java.util.concurrent`. But I have also no suggestion for a better name.

Reply:
I have thought about this when I recognized the equal names. Since out ExecutionException is basically thrown within Antenna, in which we do not use java.util.concurrent, imho the name clash should not be that important, because it should not happen in the same code. If you are using an IDE, the IDE typically supports distinguishing the two different types.